### ISSUE TITLE
macOS: libluajit missing

### ISSUE THREAD

**Issue by andrascsibi** [uspgamedev/luasteam#11]
I have trouble bundling my Love2D game on macOS. I include `luasteam.so` and then

```Error: error loading module 'luasteam' from file './luasteam.so':
	dlopen(./luasteam.so, 6): Library not loaded: /usr/local/opt/luajit/lib/libluajit-5.1.2.dylib
  Referenced from: /Users/andras/dev/terraforming-earth.git/luasteam.so
  Reason: image not found
```

Of course, if I install luajit with homebrew, it works fine, but I can't expect players to do so. 

Love2d already includes `luajit 2.0.5`, so there shouldn't be a problem theoretically, we're just looking for it in the wrong place? What am I doing wrong?

**Comment by yancouto** [comment]
Yes, I think we're having this problem as well on our game. I didn't have time to investigate yet.

**Comment by yancouto** [comment]
Fixed this in 896f77fe271348f80d33056724a4d78da172c452
